src/main/jhipster/src/main/webapp/app/shared/feedback/feedback-dialog.component.ts

```diff
@@ -26,8 +28,15 @@ export class FeedbackDialogComponent implements OnInit {
                 private feedbackService: FeedbackService,
                 private alertService: AlertService,
                 private translateService: TranslateService,
+                private languageService: JhiLanguageService,
                 private renderer2: Renderer2,
+                private route: ActivatedRoute,
                 @Inject(DOCUMENT) private _document) {
+        if (!this.feature) {
+            this.route.queryParams.subscribe((value) => {
+                this.feature = value.feature;
+            });
```

Comment:
since it's a one time thing you can also use query params snapshot
```typescript
this.feature = this.route.snapshot.queryParams.feature;
```

Reply:
Done!